doc.go

```diff
@@ -0,0 +1,142 @@
+/*
+Package thanos is a set of components that
+can provide highly available Prometheus
+setup with long term storage capabilities.
+
+See https://github.com/improbable-eng/thanos/blob/master/docs/getting_started.md for getting start.
```

Comment:
```suggestion
See https://github.com/improbable-eng/thanos/blob/master/docs/getting_started.md for first steps.
```
("see getting started for getting started" is bit a duplication)